src/vector/backends/numpy.py

```diff
@@ -429,8 +443,11 @@ class AzimuthalNumpyXY(AzimuthalNumpy, AzimuthalXY, GetItem, FloatArray):  # typ
 
     ObjectClass = vector.backends.object.AzimuthalObjectXY
     _IS_MOMENTUM = False
+    dtype: numpy.dtype[typing.Any] = numpy.dtype([("x", float), ("y", float)])
```

Comment:
For some reason, numpy does not reduce the `dtype` of a vector while obtaining a particular coordinate. The default value ensures that the correct `dtype` is used if a user does not explicitly specify one (or during internal transformations).

For example, the result of `vec3d_xyz.azimuthal` has a `dtype` of `[("x", float), ("y", float), ("z", float)]` if this default `dtype` is not set.

Reply:
Okay, but maybe we should be explicit that this is `np.float64`, rather than `float`? (It maps to the same thing, but we're relying on NumPy to do that.)

Reply:
I see, thanks!